### PR TITLE
Created basic files for project

### PR DESCRIPTION
**Pull request by janelleany**
* index.html
* styles.css
* main.js

@smith-taher @paulwooan please review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Rev My $</title>
+
+    <link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (smith-taher) [inline index.html:9]: thank you for adding the css link

File: index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+</head>
+
+<body>
+    <div class="header">
+        <h1>Rev My $</h1>
+    </div>
+    
+    <h3>Hello, yourName</h3>
+    
+    <script src="main.js"></script>
```
Comment (smith-taher) [inline index.html:20]: thank you for adding the js link

**Review by smith-taher — APPROVED** [review]
Here you go

**Review by paulwooan — APPROVED** [review]
Good work.

**Comment by janelleany** [comment]
Test

**Pull request merged**